src/sass/formio.form.scss

```diff
@@ -1035,3 +1035,9 @@ body.formio-dialog-open {
 .formio-tagpad-form-title {
   font-size: 1.7em;
 }
+
+.page-item.seen .page-link {
+  color: #fff;
+  background-color: #17a2b8;
```

Comment:
This is pretty opinionated. If you wish to have custom styles then you can include your own CSS classes that handle these types of styles. The other change you made I am fine with, but we probably would not be able to accept this since it may conflict with other bootstrap themes.

Reply:
@travist Updated the CSS class name from bootstrap defaults to formio customs.